### ISSUE TITLE
Wrong placed Class brackets are not detected

### ISSUE THREAD

**Issue by rikvdh** [cakephp/cakephp-codesniffer#31]
e.g.

``` php
class Test extends Object {
}
```

and

``` php
class Test extends Object
{
}
```

are both valid. :)
